### PR TITLE
Added: Display option for "My labels"

### LINKED ISSUE #5882: My labels can't be toggled in article display settings
> You could do that, but it's probably a lot easier in settings → display → disable article tags. ;-)

it's still visible after removing tickmarks from the settings on your screenshot above

_Originally posted by @igor17721 in https://github.com/FreshRSS/FreshRSS/issues/5869#issuecomment-1811649718_

### PR DESCRIPTION
**Pull request by math-GH**
Closes #5882

How to test the feature manually:

1. toggle the checkbox in Display configs
2. see the "my labels" in article footer in normal/global view

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/i18n/fr/conf.php [hunk 1]
```diff
@@ -41,6 +41,7 @@
 			'bottom_line' => 'Ligne du bas',
 			'display_authors' => 'Auteurs',
 			'entry' => 'Icônes d’article',
+			'myLabels' => 'My labels',	// TODO
```
Comment (Alkarex) [inline app/i18n/fr/conf.php]:
```suggestion
			'myLabels' => 'Mes étiquettes',
```

We already have the translations for that string, though

Comment (math-GH) [reply app/i18n/fr/conf.php]: yes, it is still in draft ;)

**Comment by math-GH** [comment]
Why is this PR still in draft?
We need to discuss a bit here.
The toggle is easy. But it could be also the complete disabling of the label features.

If there are 0 labels and the new checkbox is disabled, then...
This list will keep empty "forever", because only with the "add label" feature articles could be tagged
grafik

The label management could be obsolote, if there are 0 labels
grafik

**Comment by Alkarex** [comment]
Indeed. Good points @math-GH . I do not have strong feelings on that, but I would rather limit the addition of options, and it would indeed give some inconsistencies

**Comment by math-GH** [comment]
Lets merge it as it is.
The labels section in the normal view: I checked it. It is already solved. 0 labels = "My labels" will not shown.
The label management needs an bigger improvement...

**Review by Frenzie — APPROVED** [review]
lgtm, though as stated can't we just reuse the existing "My labels"?

**Pull request merged**